client/sniffer/build.gradle

```diff
@@ -100,3 +100,14 @@ thirdPartyAudit.excludes = [
   'javax.servlet.ServletContextEvent',
   'javax.servlet.ServletContextListener'
 ]
+
+javadoc {
+  dependsOn ':client:rest:javadoc'
```

Comment:
I think they can be built in any order, but if you do gradle :client:sniffer:javadoc it will run :client:rest:javadoc before that?